x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -296,4 +370,136 @@ describe('When on the Trusted Apps Page', () => {
       });
     });
   });
+
+  describe('and there are no trusted apps', () => {
+    const releaseExistsResponse: jest.MockedFunction<
+      () => Promise<GetTrustedListAppsResponse>
+    > = jest.fn(async () => {
+      return {
+        data: [],
+        total: 0,
+        page: 1,
+        per_page: 1,
+      };
+    });
+    const releaseListResponse: jest.MockedFunction<
+      () => Promise<GetTrustedListAppsResponse>
+    > = jest.fn(async () => {
+      return {
+        data: [],
+        total: 0,
+        page: 1,
+        per_page: 20,
+      };
+    });
+
+    beforeEach(() => {
+      // @ts-ignore
```

Comment:
:santa: sees you when you're coding. He knows when you're `@ts-ignore`

Reply:
🤣 
YOU MUST do all code reviews on my PRs. I'm rolling over here.